ExtraDry/ExtraDry.Blazor/Models/PropertyDescription.cs

```diff
@@ -93,7 +93,18 @@ public PropertyDescription(PropertyInfo property)
 
     public PropertyInfo Property { get; set; }
 
-    public PropertySize Size { get; set; }
+    /// <summary>
+    /// The calculated size of the property.
+    /// </summary>
+    public PropertySize Size { get; private set; }
+
+    /// <summary>
+    /// The amount of the line that the property should consume.
+    /// </summary>
+    /// <remarks>
+    /// Takes into account any consumer overrides.
+    /// </remarks>
+    public PropertySize Length => (InputFormat is not null && InputFormat.Size != PropertySize.Calculated) ? InputFormat.Size : Size;

```

Comment:
Using length as a synonym for Size is a bit confusing, especially as I think of this more as number of characters.  From the `PropertyDescription.Size`, isn't it enough to just return the resolved size?  When the PropertyDescription is created, there should be enough information to set this resolved value and never have to change it or re-calculate it.

Reply:
This property has been removed in favour of the `Size` property.